src/ui/trigger_editor.ui

```diff
@@ -704,7 +704,7 @@
                  <item>
                   <widget class="QToolButton" name="button_toggleSearchAreaResults">
                    <property name="toolTip">
-                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggles the display of the search results area.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                    <string>Toggles the display of the search results area.</string>
```

Comment:
The _nice_ wrapped/compact display is the result of using HTML to format the tooltip and __is what I would recommend gets used everywhere__ losing it like that was a `meld`-misfortune :roll_eyes: